Insights/Account/UserAccessPermissions.yaml

```diff
@@ -0,0 +1,46 @@
+SchemaVersion: '1.0'
+DataTypes:
+  - DataType: UserAccessAnalytics
+Provider: Sentinel
+Type: KQL
+EntitiesFilter: {}
+BaseQuery: |
+  let GetUserSubscriptionRbac = (v_Account_AadUserId:string) { 
+    let blastRadTime = now();
+    let userAccess = UserAccessAnalytics|where SourceEntityId == v_Account_AadUserId; 
+    let userSubAccess = userAccess| where TargetEntityType == 'AzureSubscription'| project SourceEntityName, UserId = SourceEntityId, TimeGenerated , SubscriptionName = TargetEntityName, Subscription = TargetEntityId, Role = AccessLevel, GroupId = '',GroupName= '', ServicePrincipalId = '', AccessType | summarize arg_max(TimeGenerated, *) by Subscription, Role;
+    let userGroupAccess = userAccess| where TargetEntityType == 'Group'| project SourceEntityName, UserId = SourceEntityId, GroupId = TargetEntityId, TimeGenerated, AccessType, GroupName = TargetEntityName | summarize arg_max(TimeGenerated, *) by GroupId;
+    let userGroupSubAccess = userGroupAccess | join kind = inner UserAccessAnalytics on $left.GroupId == $right.SourceEntityId | where TargetEntityType == 'AzureSubscription' and TimeGenerated <= blastRadTime | project SourceEntityName, UserId, GroupId, ServicePrincipalId = '', TimeGenerated, SubscriptionName = TargetEntityName, Subscription = TargetEntityId, Role = AccessLevel, GroupName | summarize arg_max(TimeGenerated, *) by GroupId, Subscription, Role;
+    let userSPAccess = userAccess| where TargetEntityType == 'ServicePrincipal' | project SourceEntityName, UserId = SourceEntityId, ServicePrincipalId = TargetEntityId, TimeGenerated | summarize arg_max(TimeGenerated, *) by ServicePrincipalId;
+    let userSPSubAccess = userSPAccess | join kind = inner UserAccessAnalytics on $left.ServicePrincipalId == $right.SourceEntityId | where TargetEntityType == 'AzureSubscription' and TimeGenerated <= blastRadTime | project SourceEntityName, UserId, GroupId = '', ServicePrincipalId, TimeGenerated, SubscriptionName = TargetEntityName, Subscription = TargetEntityId, Role = AccessLevel, AccessType| summarize arg_max(TimeGenerated, *) by ServicePrincipalId, Subscription, Role; 
+    userGroupSubAccess | union kind=outer userSubAccess | union kind=outer userSPSubAccess | extend Role = iff(Role == '_UNKNOWN_RBAC_ROLE_', 'Other', Role) }; 
```

Comment:
Two items:

1. The union doesn't need to be two statements, can be union kind=outer userSubAccess, userSPSubAccess.
2. Why write in Other for the unknown rbac role, I think it is more important to match the log in this case as we are not group a bunch of items to Other, just simply renaming?

Reply:
do not understand (2)